docs/stable/index.html

```diff
@@ -429,7 +429,7 @@ <h1>PyTorch documentation<a class="headerlink" href="#pytorch-documentation" tit
 <p class="caption"><span class="caption-text">Other Languages</span></p>
 <ul>
 <li class="toctree-l1"><a class="reference external" href="https://pytorch.org/cppdocs/">C++ API</a></li>
-<li class="toctree-l1"><a class="reference internal" href="packages.html">Javadoc</a></li>
+<li class="toctree-l1"><a class="reference internal" href="https://pytorch.org/javadoc/">Javadoc</a></li>
```

Comment:
We shouldn't update stable/index.html (because that should get auto generated and will be overwritten if/when we regenerate the stable docs)